tasks/main.yml

```diff
@@ -21,6 +21,7 @@
   with_items:
     - d: '{{couchdb_etc_pki_dir}}'
     - d: '{{couchdb_dir}}/etc/local.d'
+    - d: '{{couchdb_data_dir}}'
```

Comment:
Creation of the data directory should happen at init time rather than at image build time.